go.mod

```diff
@@ -1,6 +1,6 @@
-module app
+module github.com/akellbl4/WoD/app
```

Comment:
The proper name with such URL would be without /app at the end, and it better be lower case. [Here is an article about it](https://blog.golang.org/package-names).